pkg/feature/builder.go

```diff
@@ -175,36 +171,37 @@ func (fb *featureBuilder) OnDelete(cleanups ...Action) *featureBuilder {
 	return fb
 }
 
-// Load creates a new Feature instance and add it to corresponding FeaturesHandler.
+// Create creates a new Feature instance and add it to corresponding FeaturesHandler.
 // The actual feature creation in the cluster is not performed here.
-func (fb *featureBuilder) Load() error {
-	feature := newFeature(fb.name)
+func (fb *featureBuilder) Create() (*Feature, error) {
+	f := &Feature{
+		Name:    fb.featureName,
+		Managed: fb.managed,
+		Enabled: true,
```

Comment:
could be a case to disable a feature upon creation?

Reply:
The whole `Enabled` thing was stubbed in the original implementation which came with ["KServe Automated Install" initial work](https://github.com/opendatahub-io/opendatahub-operator/pull/691).

With #1057 it is finally put to use and as it was intended to be.